src/lib/isMobilePhone.js

```diff
@@ -161,6 +161,7 @@ const phones = {
   'ar-EH': /^(\+?212|0)[\s\-]?(5288|5289)[\s\-]?\d{5}$/,
   'fa-AF': /^(\+93|0)?(2{1}[0-8]{1}|[3-5]{1}[0-4]{1})(\d{7})$/,
   'mk-MK': /^(\+?389|0)?((?:2[2-9]\d{6}|(?:3[1-4]|4[2-8])\d{6}|500\d{5}|5[2-9]\d{6}|7[0-9][2-9]\d{5}|8[1-9]\d{6}|800\d{5}|8009\d{4}))$/,
+  'fr-GN': /^(\+?224|0)(6[0-356]\d{7}|722\d{6})$/,
```

Comment:
Can you move this to the correct alphabetical place between `fr-GF` and `fr-GP`?